# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../grub/default-menu.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- grub-0.97/stage2/asm.S.vanilla	2008-08-20 13:13:44.000000000 +0200
+++ grub-0.97/stage2/asm.S	2008-08-20 13:14:04.000000000 +0200
@@ -98,7 +98,7 @@
 	.string VERSION
 VARIABLE(config_file)
 #ifndef STAGE1_5
-	.string "(nd)/tftp/menu.lst"
+	.string "/boot/grub/menu.lst"
 #else   /* STAGE1_5 */
 	.long	0xffffffff
 	.string "/boot/grub/stage2"
